Mlem/App/Views/Pages/Instance/InstanceUptimeView+Views.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  InstanceUptimeView+Views.swift
+//  Mlem
+//
+//  Created by Eric Andrews on 2025-05-09.
+//
+
+import SwiftUICore
+
+struct RecentUptimeChecks: View {
+    @Environment(\.accessibilityDifferentiateWithoutColor) var diffWithoutColor: Bool
+    
+    let results: [UptimeResponseTime]
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: 3) {
+            HStack(spacing: 3) {
+                ForEach(results) { result in
+                    if diffWithoutColor {
+                        Image(icon: result.success ? .uptime.online : .uptime.offline)
+                            .resizable()
+                            .aspectRatio(contentMode: .fit)
+                            .symbolVariant(.circle.fill)
+                            .foregroundStyle(result.success ? .themedPositive : .themedNegative)
+                            .frame(maxWidth: 20)
+                            .frame(maxWidth: 25)
+                    } else {
+                        Circle()
+                            .fill(result.success ? .themedPositive : .themedNegative)
+                            .frame(maxWidth: 20)
+                            .frame(maxWidth: 25)
+                    }
```

Comment:
We can avoid some duplicate code here

(I realize you probably just copy/pasted this from wherever we had it before)

```suggestion
                Group {
                    if diffWithoutColor {
                        Image(icon: result.success ? .uptime.online : .uptime.offline)
                            .resizable()
                            .aspectRatio(contentMode: .fit)
                            .symbolVariant(.circle.fill)
                    } else {
                        Circle()
                    }
                }
                 .foregroundStyle(result.success ? .themedPositive : .themedNegative)
                .frame(maxWidth: 20)
                .frame(maxWidth: 25)
```